HealthcareApp/src/HealthcareApi/Controllers/AppointmentController.cs

```diff
@@ -0,0 +1,73 @@
+using Healthcare.Application.Appointments.Commands.Book;
+using Healthcare.Application.Appointments.Commands.Complete;
+using Healthcare.Application.Appointments.Commands.Reject;
+using Healthcare.Application.Appointments.Queries;
+using Healthcare.Appointments.Commands.Submit;
+using MediatR;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HealthcareApi.Controllers;
+
+[Route("api/[controller]s")]
+public class AppointmentController : Controller
+{
+    private readonly IMediator _mediator;
+
+    public AppointmentController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [HttpPost("request")]
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Roles = "Patient")]
```

Comment:
Define the auth schema on controller level to avoid code duplication.
Move all roles to constants to avoid potential typos